flink-runtime/src/main/java/org/apache/flink/runtime/metrics/groups/TaskManagerMetricGroup.java

```diff
@@ -81,61 +79,33 @@ public String taskManagerId() {
     //  job groups
     // ------------------------------------------------------------------------
 
-    public TaskMetricGroup addTaskForJob(
-            final JobID jobId,
-            final String jobName,
-            final JobVertexID jobVertexId,
-            final ExecutionAttemptID executionAttemptId,
-            final String taskName,
-            final int subtaskIndex,
-            final int attemptNumber) {
+    public TaskManagerJobMetricGroup getOrCreateTaskManagerJobMetricGroup(
```

Comment:
this could be named `addJob` to be consistent with the `JobManagerMetricGroup`, and, well, pretty much all other methods? :D

Reply:
Ok...though it looks like there isn't much consistency:)
```
// TaskMetricGroup
    public InternalOperatorMetricGroup getOrAddOperator(String operatorName) {
        return getOrAddOperator(OperatorID.fromJobVertexID(vertexId), operatorName);
    }
```

Reply:
ha! fair point.